src/styles.ts

```diff
@@ -172,7 +178,7 @@ export const styles = css`
 		line-height: 20px;
 		overflow: hidden;
 		text-overflow: clip;
-		position: absolute;
+		position: var(--cosmoz-input-error-message-position, absolute);
```

Comment:
What is the purpose of this var?

Reply:
I added it when I was trying to figure out a way to have the error message displayed in the input. I am cancelling it since we are keeping that the way it was before.